src/composables/date-composable.ts

```diff
@@ -0,0 +1,379 @@
+import { computed } from 'vue'
+import { isDate } from 'lodash'
+import { ApiDateTimeUtc, IsoDatePacific } from '@bcrs-shared-components/interfaces'
+import { useStore } from 'vuex'
+
+const store = useStore()
+const MS_IN_A_DAY = (1000 * 60 * 60 * 24)
+
+/** Composable that provides some useful date utilities. */
+export const DateComposable = () => {
+  const getCurrentJsDate = computed(() => store.getters['getCurrentJsDate'] as Date)
+
+  /**
+   * Fetches and returns the web server's current date (in UTC).
+   * Used to bypass the user's local clock/timezone.
+   * Ref: https://www.npmjs.com/package/serverdate
+   * @returns a promise to return a Date object
+   */
+  const getServerDate = async (): Promise<Date> => {
```

Comment:
Do we have to use this function format or can we use this?
```
async function getServerDate (): Promise<Date> {
 ...
}
```
I think this ^^^ is easier to read. What do you think?

Reply:
The function format is better. I agree.